src/utils/helpers/queryHelper.js

```diff
@@ -0,0 +1,415 @@
+import { sequelize } from '../../database/models';
+
+const tripSearchByName = async (userRole, name, user) => {
+  const search = name.toUpperCase();
+  if (userRole === 2) {
```

Comment:
userRole === 2 is confusing, use getUserRole to retrieve role name, instead of using numbers